packages/remix-template/app/routes/index.tsx

```diff
@@ -27,6 +35,18 @@ export default function Index() {
           </a>
         </li>
       </ul>
+      <p>
+        The Compute@Edge Adapter for Remix is provided as part of <a target="_blank" href="https://developer.fastly.com/labs/">Fastly Labs</a>&mdash;where Edge computing meets the cutting edge.
```

Comment:
Would suggest a colon over the em-dash here, and lowercase _Edge_.

Reply:
ok